src/clientManager.js

```diff
@@ -14,9 +14,9 @@ const initializeClient = async ({log = false, logDetailed = false, createDB, eth
 
     if (createDB) {
         try {
-            await apis[0].createDB();
+            await apis[0].createDB().timeout(10000);
```

Comment:
As a best practice, this should be configurable.

Reply:
Changed